src/spec-node/containerFeatures.ts

```diff
@@ -111,10 +111,9 @@ export async function extendImage(params: DockerResolverParameters, config: Subs
 	cliHost.mkdirp(emptyTempDir);
 	args.push(
 		'--target', featureBuildInfo.overrideTarget,
-		'-t', updatedImageName,
-		...additionalImageNames.map(name => ['-t', name]).flat(),
-		'-f', dockerfilePath,
-		emptyTempDir
+			'-f', dockerfilePath,
+		emptyTempDir,
```

Comment:
Should the context folder go after all options? (Not sure if it works the other way, but it might be safer to keep it last for compatibility.)